src/style.css

```diff
@@ -0,0 +1,150 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+
+  --color-main-text: #060b35;
+  --color-add-text: #616070;
+  --color-accent: #00acdc;
+  --color-light: #f3f3f3;
+  --main-radius: 5px;
+}
+
+.page__content {
+  margin: 0;
+  padding: 0;
+}
+
+.title {
```

Comment:
is it the separate BEM block? you don't need to create additional BEM blocks except the card and stars blocks